modules/apps/info/info-api/src/main/java/com/liferay/info/fields/reader/InfoItemFieldReaderTracker.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.info.fields.reader;
+
+import com.liferay.info.display.contributor.field.InfoDisplayContributorField;
+import org.osgi.annotation.versioning.ProviderType;
+
+import java.util.List;
+
+/**
+ * @author Jürgen Kappler
+ */
+@ProviderType
+public interface InfoItemFieldReaderTracker {
+
+	public List<InfoItemFieldReader> getInfoItemFieldReader(String className);
```

Comment:
I would name the method getInfoItemFieldReaders since it returns a list

Reply:
You are right, that was not on purpose. 
Thanks for finding it.